src/main/java/com/monumental/services/TagService.java

```diff
@@ -8,11 +8,36 @@
 @Service
 public class TagService extends ModelService<Tag> {
 
+    /**
+     * Public constructor for TagService
+     * Use when NOT injecting SessionFactoryService via Spring
+     * @param sessionFactoryService - instance of SessionFactoryService to use for initialization
+     */
+    public TagService(SessionFactoryService sessionFactoryService) {
```

Comment:
If all of the services are going to have this then it should just be done in the `ModelService` itself